.github/CODEOWNERS

```diff
@@ -1,2 +1,2 @@
-* @s2n-core
-/codebuild @dougch @s2n-core
+* @aws/s2n-core
+/codebuild @dougch @aws/s2n-core
```

Comment:
Should this be showing up in your diff? Did something go odd with your rebase / merge?

Reply:
The branch seems to be reapplying this PR

Reply:
I rebased on top of the wrong head